src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -1245,12 +1248,13 @@ public ResultSet getPrimaryKeys(String c, String s, String table) throws SQLExce
      */
     public ResultSet getExportedKeys(String catalog, String schema, String table)
             throws SQLException {
-        PrimaryKeyFinder pkFinder = new PrimaryKeyFinder(table);
+        PrimaryKeyFinder pkFinder = new PrimaryKeyFinder(table, schema);
         String[] pkColumns = pkFinder.getColumns();
         Statement stat = conn.createStatement();
 
         catalog = (catalog != null) ? quote(catalog) : null;
-        schema = (schema != null) ? quote(schema) : null;
+
+        String quotedSchema = (schema != null) ? quote(schema) : null;
```

Comment:
Any specific reason for this?

Reply:
We use the schema later as an identifier, it will not work with string quotations